Documentation/NvimConfiguration.md

```diff
@@ -0,0 +1,315 @@
+# NVim Project Configuration
+
+NVim can be configured to use the [COC-clangd](https://github.com/clangd/coc-clangd)
+plugin to provide code-completion as well as inline
+[git blame](https://github.com/f-person/git-blame.nvim) using [vim-plug](https://github.com/junegunn/vim-plug).
+
+Make sure you ran `./Meta/serenity.sh rebuild-toolchain` as well as
+`./Meta/serenity.sh run`.
+
+# 0. Install vim-plug
```

Comment:
I don't think it's necessary to number the headings, we don't do that anywhere else.

Reply:
Done.